src/main/java/mate/academy/internetshop/controllers/DeleteProductFromShoppingCartController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.internetshop.controllers;
```

Comment:
```suggestion
package mate.academy.internetshop.controller;
```
Пекеджі потрібно називати в однині

Reply:
Done